docs/api/tools/block-explorer/batches-view.md

```diff
@@ -0,0 +1,29 @@
+# Browse batches
+
+Batches are like groups of transactions that are processed together in one [block](../../../dev/developer-guides/transactions/blocks.md#blocks) before they're added to the main blockchain. It increases the transaction throughput of zkSync Era and reduces its transaction fees.
+
+By putting several transactions into a single batch, we reduce the number of transactions that need to be processed and validated, this results in faster transaction processing times and lower fees for users interacting with zkSync Era.
+
+![Browse batches!](../../../assets/images/batches.png "View batches on zkSync")
+
+On the block explorer home screen, you'll see the 10 most recent batches by default. If you want to see all the recent batches or get details on a specific batch, you can go to the batches page, which is also accessible from the top menu.
```

Comment:
```suggestion
On the block explorer home screen, you see the 10 most recent batches by default. If you want to see all recent batches, or details on a specific batch, go to the batches page which is accessible from the top menu.
```

Reply:
Latest batches is clear in the dashboard. Use a full dashboard pic here with arrow.